src/Uno.UI/UI/Xaml/Controls/FlipView/FlipView.managed.cs

```diff
@@ -1542,7 +1546,7 @@ internal override void OnSelectedIndexChanged(int oldSelectedIndex, int newSelec
 			m_skipAnimationOnce = false;
 
 			var smallChange = Math.Abs(oldSelectedIndex - newSelectedIndex) <= 1;
-			OnSelectedIndexChangedPartial(oldSelectedIndex, newSelectedIndex, smallChange && UseTouchAnimationsForAllNavigation);
+			//OnSelectedIndexChangedPartial(oldSelectedIndex, newSelectedIndex, smallChange && UseTouchAnimationsForAllNavigation);
```

Comment:
This looks incorrect, the method sems defined and used.

Reply:
@jeromelaban This file is only for Wasm and Skia. The method is only defined for Android and iOS